cloudformation/stacks/AutoSpotting/template.yaml

```diff
@@ -152,6 +160,19 @@
         | Linux/UNIX (Amazon VPC) | SUSE Linux (Amazon VPC) | Windows (Amazon
         VPC)"
       Type: "String"
+    StackSets:
```

Comment:
I've been thinking further and to keep things simple we may not even need to create a new custom resource, but just extend the existing Python inline Lambda function code to query the ARN of the main Lambda function at runtime, assuming it has enough permissions from the IAM role.

We just need to somehow tag the function from the current stackset so we're able to find it in case there are more AutoSpotting installations in the account.

In case this python code gets large enough we may want to split it into an additional file, but that may complicate some things a bit.

Reply:
@mello7tre are you working on this? I may be able to take it over as I have some bandwidth these days.

Please share if you had any work in progress towards it.

Reply:
Sorry @cristim but i am pretty busy at work, and had no time to work on this.
If you have ideas and time, feel free to take it over.

Just one thing, i also think that splitting the lambda would complicate further and i would not go in that direction.

Reply:
Hi @cristim, just to inform you that i re-begin working on this and i think i am near to have  a working solution.

As soon as i have a working template i will push the commit.